### ISSUE TITLE
Uncaught SyntaxError: Failed to execute 'addColorStop' on 'CanvasGradient': The value provided ('rgba(NaN, NaN, NaN, 1)') could not be parsed as a color.

### ISSUE THREAD

**Issue by moerabaya** [thomaslindstrom/color-picker#112]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.209.0
**System**: Mac OS X 10.10.3
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v2.0.4

### Stack Trace

Uncaught SyntaxError: Failed to execute 'addColorStop' on 'CanvasGradient': The value provided ('rgba(NaN, NaN, NaN, 1)') could not be parsed as a color.

```
At /Users/mohammedrabaya/.atom/packages/color-picker/lib/extensions/Alpha.coffee:100

Error: Failed to execute 'addColorStop' on 'CanvasGradient': The value provided ('rgba(NaN, NaN, NaN, 1)') could not be parsed as a color.
    at Error (native)
    at Object._this.canvas.render (/Users/mohammedrabaya/.atom/packages/color-picker/lib/extensions/Alpha.coffee:89:27)
    at /Users/mohammedrabaya/.atom/packages/color-picker/lib/extensions/Alpha.coffee:96:35
    at Object.emit (/Users/mohammedrabaya/.atom/packages/color-picker/lib/modules/Emitter.coffee:15:21)
    at Object.emitColorChanged (/Users/mohammedrabaya/.atom/packages/color-picker/lib/extensions/Saturation.coffee:15:29)
    at /Users/mohammedrabaya/.atom/packages/color-picker/lib/extensions/Saturation.coffee:194:33
    at Object.emit (/Users/mohammedrabaya/.atom/packages/color-picker/lib/modules/Emitter.coffee:15:21)
    at Object.emitSelectionChanged (/Users/mohammedrabaya/.atom/packages/color-picker/lib/extensions/Saturation.coffee:9:29)
    at Object._this.control.setSelection (/Users/mohammedrabaya/.atom/packages/color-picker/lib/extensions/Saturation.coffee:181:35)
    at /Users/mohammedrabaya/.atom/packages/color-picker/lib/extensions/Saturation.coffee:191:36
```

### Commands

```
     -9:09.9.0 color-picker:open (input.hidden-input)
  2x -9:03.2.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
  2x -1:26.8.0 color-picker:open (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
     -1:09.7.0 core:confirm (input.hidden-input)
     -0:04.9.0 core:copy (div.package-readme.native-key-bindings)
     -0:03.7.0 core:paste (input.hidden-input)
     -0:02.3.0 color-picker:open (input.hidden-input)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
color-picker, v2.0.4

# Dev
No dev packages
```

**Comment by moerabaya** [comment]
It's look like it's a system problem, just remove the package manually and install it again, it will work fine!

**Comment by thomaslindstrom** [comment]
Hi, @moerabaya. I might have fixed this in `v2.0.6`, which is probably why reinstalling worked. Thanks!
